src/rest_framework_jwt/blacklist/serializers.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from datetime import datetime
+
+from django.utils.timezone import make_aware
+
+from rest_framework import serializers
+
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_jwt.blacklist.models import BlacklistedToken
+from rest_framework_jwt.settings import api_settings
+from rest_framework_jwt.utils import check_user, unix_epoch
+
+
+class BlacklistTokenSerializer(serializers.ModelSerializer):
+    """
+    Serializer used for blacklisting tokens.
+    """
+
+    class Meta:
+        model = BlacklistedToken
+        fields = ('token', )
```

Comment:
You can specify that the token is optional via [`extra_kwargs`](https://www.django-rest-framework.org/api-guide/serializers/#additional-keyword-arguments). This avoids any confusion when users are looking at automatically generated documentation (e.g. Swagger).